grommet-leaflet/src/layers/Map/index.d.ts

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import * as L from 'leaflet';
+
+export interface MapProps {
+  center?: number[];
```

Comment:
center, scrollWheelZoom, zoom, and zoomControl all from from Leaflet. Can we remove them here since we extend L.MapOptions?

Reply:
done